src/Translator.php

```diff
@@ -30,6 +30,7 @@ public function __construct(string $locale)
                 'till_hour' => 'till',
                 'closed' => 'closed',
                 'open' => 'open at',
+                'open_from' => 'open at',
```

Comment:
This confuses me, the English key is `open_from` but the English translation is `open at`. Would it be more clear to use `open_at` as key?

Reply:
Maybe the first key should be 'open_at' & the second 'open_at_multiple_days'? cause the from is only for the Dutch translation?

Reply:
That is even better yes.